### DISCUSSION TITLE
[QUESTION] Add "Private" Button function to Custom module

### DISCUSSION THREAD

**Discussion by kseibi** [YetiForceCompany/YetiForceCRM#14612] [How to]
Hi, my question might be a little bit silly, but I cannot figure out how to add "Private" checkbox button on a new custom module, when adding the field as a checkbox it says that "Field name is reserved", where can I add this field? thanks

Yetiforce 6.1 Version

**Comment by rskrzypczak** [comment]
Private field is a special field type and cannot be added from GUI.
The field will be added by default to custom modules from v6.2
Check this: https://github.com/YetiForceCompany/YetiForceCRM/commit/767d22e5baa00b5d95401ad7bf2bccbc5e1bd534
